client/my-sites/customer-home/cards/features/grow-earn/index.jsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Card } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import CardHeading from 'components/card-heading';
+import VerticalNav from 'components/vertical-nav';
+import VerticalNavItem from 'components/vertical-nav/item';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { SIDEBAR_SECTION_TOOLS } from 'my-sites/sidebar/constants';
+import { expandMySitesSidebarSection as expandSection } from 'state/my-sites/sidebar/actions';
+import { bumpStat, composeAnalytics, recordTracksEvent } from 'state/analytics/actions';
+import { getSelectedEditor } from 'state/selectors/get-selected-editor';
+import { getSiteOption } from 'state/sites/selectors';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export const GrowEarn = ( { siteSlug, expandToolsAndTrack } ) => {
+	const translate = useTranslate();
+
+	return (
+		<Card className="grow-earn">
+			<CardHeading>{ translate( 'Grow & Earn' ) }</CardHeading>
+			<h6 className="grow-earn__card-subheader">
```

Comment:
We could also keep using `customer-home__card-subheader` here so we don't need to copy shared styles across all components.

To get rid of the linter issue in that case, we can use both classes: `grow-earn__card-subheader customer-home__card-subheader`.

Reply:
> To get rid of the linter issue in that case, we can use both classes: grow-earn__card-subheader customer-home__card-subheader.

Ah, I didn't even think of that! I admit to quietly bristling every time we change code to something less optimal/accurate for the sake of a digital overlord, but that's a reasonable-ish solution (still silly, but better than duping styles). 👍

Reply:
9d52b0e28d